docs/TEST_SCENARIOS.md

```
@@ -0,0 +1,585 @@
+
+## 3. Token Validation
+
+### Scenario 3.1: Successful token validation (requires valid FreeFeed token)
+
+**Preconditions**:
+- User is signed in
+- Token created with status "Pending"
+
+**Steps**:
+1. Copy the displayed token value
+2. Open FreeFeed in new tab (https://freefeed.net)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)